src/plugins/intel_gpu/src/plugin/ops/interpolate.cpp

```diff
@@ -50,18 +50,22 @@ static void CreateInterpolateOp(Program& p, const std::shared_ptr<ngraph::op::v4
     if (axes.size() != scales.size())
         IE_THROW() << op->get_friendly_name() << " Incorrect axes and scales should be the same size";
 
+    // TODO shouldn't be all this checking done in ngraph::op::v4::Interpolate?
```

Comment:
It needs to discuss how to handle axes because sometimes the axes are inserted incorrectly at the model optimizer. But, if it works well with unit tests, I think it is enough. You can refer to CPU plugin. [Interpolate](https://github.com/openvinotoolkit/openvino/blob/6d6f52806bdddd6e2c49ffe8f39da87eb034bf75/src/plugins/intel_cpu/src/nodes/interpolate.cpp#L1547)

Reply:
then I take away this comment.